File: Comparativa Python/Ejercicio11-Condicionales.py
# 11. Hacer un programa que simule un cajero automático con un saldo inicial
# de 1000 Dólares.

saldoInicial = 1000.00

print('\tBienvenido a su cajero virtual')
print('1. Ingresar dindero en cuenta')
print('2. Retirar dindero en cuenta')
print('3. Salir')
opcion = int(input('Digite el número deseado [1-3]: '))

if opcion == 1:
    cantidad = float(input('Digite cantidad a depositar: '))
    saldo = saldoInicial + cantidad
    print('Dinero en cuenta: $', str(round(saldo, 2)))
elif opcion == 2:
    cantidad = float(input('Digite cantidad a retirar: '))
    if cantidad > saldoInicial:
        print('Saldo insuficiente')
    else:
        saldo = saldoInicial - cantidad
        print('Dinero en cuenta: $', str(round(saldo, 2)))
elif opcion == 3:
    print('Fin de programa')
else:
    print('Valor fuera de rango')

File: Ejercicio11-Condicionales.cpp
/* 11. Hacer un programa que simule un cajero automático con un saldo inicial de 1000 Dólares.*/

#include<iostream>
using namespace std;
int main(){
    int  saldo_inicial = 1000, opc;
    float extra, saldo = 0, retiro;

    cout<<"\tBienvenido a su cajero virtual"<<endl;
    cout<<"1. Ingresar dinero en cuenta"<<endl;
    cout<<"2. Retirar dinero de la cuenta"<<endl;
    cout<<"3. Salir"<<endl;
    cout<<"Opcion: ";
    cin>>opc;

    switch(opc){
        case 1:
            cout<<"Digite la cantidad de dinero a ingresar: ";
            cin>>extra;
            saldo = saldo_inicial + extra;
            cout<<"Dinero en cuenta: $"<<saldo;break;

        case 2:
            cout<<"Digite la cantidad de dinero que va a retirar: ";
            cin>>retiro;

            if (retiro > saldo_inicial){
                cout<<"No tiene esa cantidad de dinero";
            }
            else{
                saldo = saldo_inicial - retiro;
                cout<<"Dinero en cuenta: $"<<saldo; break;
            }
        case 3: break;
    }
    return 0;
}